src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -35,7 +36,7 @@ class Yaml
      * @param string $input                  A string containing YAML
      * @param bool   $exceptionOnInvalidType True if an exception must be thrown on invalid types false otherwise
      * @param bool   $objectSupport          True if object support is enabled, false otherwise
-     * @param bool   $objectForMap           True if maps should return a stdClass instead of array()
+     * @param bool   $objectForMap           True if maps should return an stdClass instead of array()
```

Comment:
In general we put "a" in front of a consonant. It has more to do how it's pronounced rather then how it's written though.